Xamarin.Forms.Platform.Android/Renderers/ShellBottomNavViewAppearanceTracker.cs

```diff
@@ -80,7 +81,9 @@ protected virtual void SetBackgroundColor(BottomNavigationView bottomView, Color
 				var child = menuView.GetChildAt(index);
 				var touchPoint = new Point(child.Left + (child.Right - child.Left) / 2, child.Top + (child.Bottom - child.Top) / 2);
 
-				bottomView.SetBackground(new ColorChangeRevealDrawable(_lastColor.ToAndroid(), color.ToAndroid(), touchPoint));
+				_colorChangeRevealDrawable = new ColorChangeRevealDrawable(_lastColor.ToAndroid(), color.ToAndroid(), touchPoint);
+
+				bottomView.SetBackground(_colorChangeRevealDrawable);
```

Comment:
maybe this should be set to null on dispose?

Reply:
I changed my approach.
I now only dispose the old background after it has been changed.
The final dispose is done in ShellItemRenderer.
I succeed to reproduce when disposing the AppearanceTracker before the bottom view background. But in the current order it work fine. So not sure if it will work.